nixos-generate

```diff
@@ -35,6 +36,7 @@ Options:
 * --system: specify the target system (eg: x86_64-linux)
 * --cores : to control the maximum amount of parallelism. (see nix-build documentation)
 * -I KEY=VALUE: Add a key to the Nix expression search path.
+* --nixpkgs_ver: set the release version of nixpkgs
```

Comment:
I don't know if it's worth to have yet another shortcut. You get the same by using the `-I` argument. Eg: `-I nixpkgs=channel:nixos-19.09`

Reply:
your right, wrote yesterday with lassulus on matrix about it.
a hint in the README.md is enough at that point, i think.

Reply:
your right at that point. will correct that.

Reply:
good idea